src/components/ProjectCard.tsx

```diff
@@ -126,10 +129,16 @@ export default function ProjectCard({
   return (
     <Link href={projectCardHref} as={projectCardUrl}>
       <a>
-        <div className="relative flex cursor-pointer items-center overflow-hidden whitespace-pre rounded-sm border border-solid border-smoke-300 py-6 px-5 transition-colors hover:border-smoke-500 dark:border-slate-300 dark:hover:border-slate-100">
+        <div
+          className={`relative flex cursor-pointer items-center overflow-hidden whitespace-pre ${
+            isMobile
+              ? 'py-4'
+              : 'rounded-sm border border-solid border-smoke-300 py-6 px-5 transition-colors hover:border-smoke-500 dark:border-slate-300 dark:hover:border-slate-100'
```

Comment:
**suggestion** we can use tailwind media queries for all this. can slap `md:<class>` for all of the false case of the turnary.